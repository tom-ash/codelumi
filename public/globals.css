:root {
  --background: #ffffff;
  --foreground: #171717;
  --error: #e74c3c;
}

html,
body {
  max-width: 100vw;
  overflow-x: hidden;
}

body {
  /* color: var(--foreground);
  background: var(--background); */
  font-family: var(--font-geist-sans);
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

* {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}

/* Remove arrows in Chrome, Safari, Edge, and Opera */
input[type='number']::-webkit-inner-spin-button,
input[type='number']::-webkit-outer-spin-button {
  -webkit-appearance: none;
  margin: 0;
}

/* Remove arrows in Firefox */
input[type='number'] {
  -moz-appearance: textfield;
}

/* a {
  color: inherit;
  text-decoration: none;
} */

/* @media (prefers-color-scheme: dark) {
  html {
    color-scheme: dark;
  }
} */

/* 

@media (prefers-color-scheme: dark) {
  :root {
    --background: #0a0a0a;
    --foreground: #ededed;
  }
} */
