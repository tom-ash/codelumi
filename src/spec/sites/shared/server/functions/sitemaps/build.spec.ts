import { describe, it } from '@jest/globals';
import buildSitemap from '../../../../../../sites/shared/server/functions/sitemaps/build';

const clientUrl = 'https://mapawynajmu.pl'

const rootLinkGroup = [
    {
      href: '/',
      hrefLang: 'pl',
      priority: '1.0',
      changeFreq: 'daily',
    },
    {
      href: '/en',
      hrefLang: 'en',
      priority: '1.0',
      changeFreq: 'daily',
    },
  ]

describe('buildSitemap', () => {
  it('is delicious', () => {
    expect(buildSitemap({ linkGroups: [rootLinkGroup], clientUrl })).toMatchSnapshot()
  });
})
