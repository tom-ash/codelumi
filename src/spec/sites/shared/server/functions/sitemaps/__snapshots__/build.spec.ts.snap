// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buildSitemap is delicious 1`] = `
"<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
<urlset xmlns=\\"http://www.sitemaps.org/schemas/sitemap/0.9\\" xmlns:xhtml=\\"http://www.w3.org/1999/xhtml\\">
    <url>
        <loc>
            https://mapawynajmu.pl/
        </loc>
        <changefreq>
            daily
        </changefreq>
        <priority>
            1
        </priority>
        <xhtml:link rel=\\"alternate\\" href=\\"https://mapawynajmu.pl/\\" hreflang=\\"pl\\"/>
        <xhtml:link rel=\\"alternate\\" href=\\"https://mapawynajmu.pl/en\\" hreflang=\\"en\\"/>
    </url>
    <url>
        <loc>
            https://mapawynajmu.pl/en
        </loc>
        <changefreq>
            daily
        </changefreq>
        <priority>
            1
        </priority>
        <xhtml:link rel=\\"alternate\\" href=\\"https://mapawynajmu.pl/\\" hreflang=\\"pl\\"/>
        <xhtml:link rel=\\"alternate\\" href=\\"https://mapawynajmu.pl/en\\" hreflang=\\"en\\"/>
    </url>
</urlset>"
`;
